ui/components/multichain-accounts/multichain-accounts-tree/multichain-accounts-tree.test.tsx

```diff
@@ -20,292 +20,240 @@ const walletTwoId: AccountWalletId = 'entropy:01JKAF3PJ247KAM6C03G5Q0NP8';
 const walletTwoGroupId: AccountGroupId =
   'entropy:01JKAF3PJ247KAM6C03G5Q0NP8:default';
 
+const createAccount = (
+  id: string,
+  name: string,
+  address: string,
+  hidden = false,
+  pinned = false,
+  type = EthAccountType.Eoa,
+): Merge => ({
```

Comment:
Since there's a lot of parameters, we could use an object instead for the parameters, allowing us to use it that way:

```ts
createAccount({
  id: 'account-1',
  name: 'Account 1',
  address: '0x...',
  pinned: false
});
```

WDYT?